Patch-Source: https://github.com/php/php-src/commit/962c082a5b7b94f7a3779378e8ab83bedb4445ad
From 962c082a5b7b94f7a3779378e8ab83bedb4445ad Mon Sep 17 00:00:00 2001
From: Anatol Belski <ab@php.net>
Date: Sun, 2 Oct 2022 19:54:34 +0200
Subject: [PATCH] fileinfo: Fix expected values for test on s390x

The peak memory usage increased by the blanket 1mb on s390 only.
This is good to note but otherwise seems acceptable, so just
adjusting the test is fine.

Signed-off-by: Anatol Belski <ab@php.net>
---
 ext/fileinfo/tests/bug78987.phpt | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/ext/fileinfo/tests/bug78987.phpt b/ext/fileinfo/tests/bug78987.phpt
index 617d8ca4412d..423ecb1b8517 100644
--- a/ext/fileinfo/tests/bug78987.phpt
+++ b/ext/fileinfo/tests/bug78987.phpt
@@ -11,14 +11,14 @@ $minSize = 128 * 1024;
 $maxSize = 16 * 1024 * 1024;
 
 $map = array(
-	131072 => 9113600,
-	262144 => 9113600,
-	524288 => 10850304,
-	1048576 => 11210752,
-	2097152 => 13312000,
-	4194304 => 17510400,
-	8388608 => 23801856,
-	16777216 => 36384768,
+	131072 => 10055680,
+	262144 => 10055680,
+	524288 => 11898880,
+	1048576 => 12152832,
+	2097152 => 14254080,
+	4194304 => 18452480,
+	8388608 => 24743936,
+	16777216 => 37326848,
 );
 for($size = $minSize; $size <= $maxSize; $size *= 2) {
 	$content = str_repeat('0', $size);
